lib/screens/search/result_programs_list.dart

```diff
@@ -36,30 +36,37 @@ class ResultProgramsList extends StatelessWidget {
 }
 
 class _Program extends StatelessWidget {
-  final SearchResultItem item;
-  final slideWidth = 140.0;
+  final SearchResultItem _item;
+  final _slideWidth = 140.0;
 
-  const _Program(this.item);
+  const _Program(this._item);
 
   @override
   Widget build(BuildContext context) {
     return Column(
       crossAxisAlignment: CrossAxisAlignment.start,
       children: [
         Container(
-          height: 80,
-          width: slideWidth,
+          width: _slideWidth,
+          height: _slideWidth * (9 / 16),
```

Comment:
there's an aspect ratio component

Reply:
I hadn't used it because that would introduce 2 additional widgets. Because AspectRatio sets the size of the child based on the dimentions of the parent. So need to add an additional parent conatiner to set the width.

```
Container(
    width: <>,
    AspectRatio(
        ContainerWithImage()
    )
)
```
Should I still use AspectRatio?